src/current/v23.1/migration-overview.md

```diff
@@ -231,7 +227,7 @@ First, convert your database schema to an equivalent CockroachDB schema:
 
 Then import the converted schema to a CockroachDB cluster:
 
-- For CockroachDB {{ site.data.products.cloud }}, use the Schema Conversion Tool to [migrate the converted schema to a new {{ site.data.products.serverless }} or {{ site.data.products.dedicated }} database](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page#migrate-the-schema).
+- For CockroachDB {{ site.data.products.cloud }}, use the Schema Conversion Tool to [migrate the converted schema to a new {{ site.data.products.cloud }} database](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page#migrate-the-schema).
```

Comment:
Similar to feedback above about Standard cluster vs Cloud cluster.

Reply:
Same comment as above: the point is that SCT creates a db on whichever Cloud cluster you're running it on. So if someone is migrating to Cloud, we assume they know which tier they're using, or at least that this question should be separate from the migration overview. At worst, it's potentially misleading people to think that they need to use the SCT with specifically a Standard cluster.